test/e2e/upgrade/adminack/adminack.go

```diff
@@ -51,7 +55,22 @@ func (t *UpgradeTest) Test(f *framework.Framework, done <-chan struct{}, upgrade
 		cancel()
 	}()
 
-	adminAckTest := &clusterversionoperator.AdminAckTest{Oc: t.oc, Config: t.config, Poll: 10 * time.Minute}
+	pollDuration := 10 * time.Minute
+	// Run test once on SNO upgrade. We don't guarantee API availability during upgrades there
+	configClient, err := configv1client.NewForConfig(t.config)
+	if err != nil {
+		return
+	}
+	infra, err := configClient.ConfigV1().Infrastructures().Get(ctx, "cluster", metav1.GetOptions{})
```

Comment:
At this point it may already be too late, the upgrade could have already started at this point. It's a bit racey

Reply:
Hmm, perhaps I should move it to `Setup` then